types/types.go

```diff
@@ -249,7 +251,7 @@ type Image interface {
 
 // ManifestUpdateOptions is a way to pass named optional arguments to Image.UpdatedManifest
 type ManifestUpdateOptions struct {
-	LayerInfos              []BlobInfo // Complete BlobInfos (size+digest+urls) which should replace the originals, in order (the root layer first, and then successive layered layers)
+	LayerInfos              []BlobInfo // Complete BlobInfos (size+digest+urls+mediatype+annotations) which should replace the originals, in order (the root layer first, and then successive layered layers)
```

Comment:
This is not currently implemented that way.

Same as `ManifestUpdateInformation.LayerInfos` below, the URLs+MIME type + annotations are not currently available when called by `copy.Image`.

Also, the `UpdatedImage` implementations currently do not use the MIME type from this array (but they do use URLs+annotations). Changing the two lines would be trivial, but when `copy.Image` does not set the values, that would break things.

So, I guess, for now explicitly document that the MIME type is ignored?